memgpt/cli/cli.py

```diff
@@ -38,6 +38,7 @@ def run(
     human: str = typer.Option(None, help="Specify human"),
     model: str = typer.Option(None, help="Specify the LLM model"),
     preset: str = typer.Option(None, help="Specify preset"),
```

Comment:
@sarahwooders example here - technically "preset" = "system prompt" + "function set"

So there's a slight conflict allowing people to also specify "system prompt", but maybe it's OK? Just overwrite the implicit choice of system prompt in preset?

Reply:
Maybe it is better to allow users to make their custom presets with system prompt and function set? But it is out of the scope of this PR.